src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    String title;
```

Comment:
Will it be more appropriate to use 'description'?  'Title' is totally dine also.